### ISSUE TITLE
shufflenetV2 logits

### ISSUE THREAD

**Issue by awsssix** [tensorpack/tensorpack#1198]
### 1. What you did:

(1) **If you're using examples, what's the command you run:**
$python shufflenet.py --eval --data ./data --load shufflenet-v1.0.npz -r=1

(2) **If not using examples, tell us what you did:**
    After eval ,I only got top-1 and top5 error!
    But  I want to get the corresponding label of image, is logits?
![image](https://user-images.githubusercontent.com/46213799/58000232-b8184b00-7b0a-11e9-9500-12d102ce52a9.png)
    how should I change the code?

**Comment by ppwwyyxx** [comment]
See https://tensorpack.readthedocs.io/tutorial/inference.html#offlinepredictor

For shufflenet example, you can change the output names at https://github.com/tensorpack/tensorpack/blob/8879637399b665bfd11aa5ded663619a1a19bdb9/examples/ImageNetModels/imagenet_utils.py#L272-L284

You can `print(logits)` to see that its name is `linear/output`.

**Comment by awsssix** [comment]
@ppwwyyxx Thanks you very much!
I still have a question,how can i test only one pic?

**Comment by ppwwyyxx** [comment]
The example at https://tensorpack.readthedocs.io/tutorial/inference.html#offlinepredictor shows how to test on any input array.

**Comment by awsssix** [comment]
@ppwwyyxx Thank you very much, I've worked it out.
Now I only have almost 10,0000 pics for train, it is less for shufflenetV2?
And how many training sets are usually required for shufflenetV2?

**Comment by ppwwyyxx** [comment]
That's a machine learning question, so we cannot answer. I'd assume that if you have <100 classes 100000 images would be enough.

**Comment by awsssix** [comment]
@Oh ，thanks for your replay！
I’d love to know what you think  100000  images ， 300 classes, will enough for shufflenetV2?
Or do I need more training sets?
